src/main/java/io/seqera/migtool/Dialect.java

```diff
@@ -0,0 +1,60 @@
+package io.seqera.migtool;
+
+import groovyjarjarantlr4.v4.runtime.misc.Nullable;
```

Comment:
This `Nullable` annotation comes from a Groovy package, any chance of using other? In the end, importing `Nullable` is [not as trivial](https://stackoverflow.com/questions/76630457/what-nullable-to-use-in-java-as-of-2023-jdk21) as it might seem. Alternatively, we could just not use it.

Reply:
In the end, the annotation is just a marker so that people know that they should check for or expect a null value. Do we care from which package it comes from? If so, I can remove it, but probably it is ok ...

Reply:
Okay, I was saying that because it seems to come from a somewhat obscure Groovy package that may cause confusion, but I personally don't mind.

Reply:
I'll see if I can a more known one instead.